cmd/skopeo/main.go

```diff
@@ -75,3 +81,19 @@ func main() {
 		logrus.Fatal(err)
 	}
 }
+
+// getPolicyContext handles the global "policy" flag.
+func getPolicyContext(c *cli.Context) (*signature.PolicyContext, error) {
+	policyPath := c.GlobalString("policy")
+	var policy *signature.Policy // This could be cached across calls, if we had an application context.
```

Comment:
Yeah, OTOH it is easier to attach the “This could be cached” comment to the `policy` variable when it is on a separate line.